### PR TITLE
Automated OCP-19695

### PR DESCRIPTION
**Pull request by SSshahan**
@yapei please help to review, thanks

### CODE REVIEW

File: lib/rules/web/admin_console/4.4/deployment.xyaml [hunk 1]
```diff
@@ -61,3 +61,39 @@ check_sections_on_container_details:
       xpath: //h2[contains(@class,'co-section-heading') and .='Image Details']
   - selector:
       xpath: //h2[contains(@class,'co-section-heading') and .='Network']
+check_hpa_metrics_table:
+  elements:
+  - selector:
+      xpath: //div/h2[contains(@data-test-section-heading,'Metrics')]/following-sibling::div[contains(.,'Type')]/../div[contains(.,'Current')]/../div[contains(.,'Target')]
```
Comment (yapei) [inline lib/rules/web/admin_console/4.4/deployment.xyaml]: This looks a little complicated, how about this 
```
elements:
- selector:
     xpath: //h2[contains(@data-test-section-heading,'Metrics')]/..//div[contains(@class,'co-m-table-grid__head')]/div[contains(.,'Type')]
- selector:
     xpath: //h2[contains(@data-test-section-heading,'Metrics')]/..//div[contains(@class,'co-m-table-grid__head')]/div[contains(.,'Current')]
- selector:
     xpath: //h2[contains(@data-test-section-heading,'Metrics')]/..//div[contains(@class,'co-m-table-grid__head')]/div[contains(.,'Target')]

**Comment by SSshahan** [comment]
@liangxia  could you help to merge this? Thanks!

File: lib/rules/web/admin_console/4.4/deployment.xyaml (hunk 1 above)
Comment (liangxia) [reply lib/rules/web/admin_console/4.4/deployment.xyaml]: @yapei @SSshahan  Did you got agreement on this ?

Comment (SSshahan) [reply lib/rules/web/admin_console/4.4/deployment.xyaml]: @liangxia  yes, we have come to agreement on this

**Pull request merged**